### DISCUSSION TITLE
Thống nhất "premise" trong NLI

### DISCUSSION THREAD

**Discussion by cuongvng** [d2l-ai/d2l-vi#2735] [Terminology]
Hiện trong các PRs đang có 2 cách dịch là `tiền đề` và `tiên đề`. Mọi người vote nhé!

`tiên đề` 👍 
`tiền đề` 😄

cc: @lkhphuc @rootonchair @davidnvq @minhduc0711 @duythanhvn @trGiang99 @LongNguyen1984

**Comment by quangvnai** [comment]
sao a cảm giác từ `tiên đề` nghe giống từ `axiom` trong [toán](https://vi.wikipedia.org/wiki/Ti%C3%AAn_%C4%91%E1%BB%81) thế nhỉ?
Với lại a thấy từ `tiền đề` hay dịch đi kèm từ giả thuyết trong bộ premise - hypothesis. Mọi người thử google cặp "tiên đề - premise" xem.

**Reply by cuongvng** [reply]
hợp lý anh 👍

**Reply by quangvnai** [reply]
Thêm một ví dụ về suy luận sử dụng bộ [`tiền đề - giả thuyết`](http://www.vhntdng.vn/root/pages/nghien-cuu-khoa-hoc-33117293/web-dev.aspx). anh chưa làm NLI bao giờ và cũng ít gặp cặp từ này nên đành tham khảo các tài liệu có sẵn vậy, search ra cũng hơi ít tài liệu.

**Reply by LongNguyen1984** [reply]
"Tiền" và "Tiên" theo âm hán thì đều cùng chỉ là "trước".  Ví dụ trong câu "Tiên học lễ hậu học văn" thì Tiên là trước. Chữ "Tiền Tuyến", "Tiền đạo" thì Tiền cũng có nghĩa là trước

**Reply by lkhphuc** [reply]
`tiên` với `tiền` đúng là nghĩa như nhau nhưng như @davidnvq nói `tiên đề` đang hay được dịch từ `axiom/postulate` rồi, vậy nên mình cũng vote `tiền đề`.
Như ở đây họ cũng ghi rõ `tiền đề` là vế thứ nhất trong `tam đoạn luận`.
https://vi.wiktionary.org/wiki/ti%E1%BB%81n_%C4%91%E1%BB%81
